labels/en-US.json

```diff
@@ -340,6 +340,8 @@
   "friday": "Friday",
   "from": "from",
   "fromDeliveryLocation": "from location",
+	"fulfillmentDirectShip": "Direct Ship",
```

Comment:
fix indentation over here. And you also have to add this to another localization file as well apart from english.

Reply:
done

Reply:
@kibo-DeepakTajane , Based on this comment, you've missed adding these new labels into "de-DE.json". see my original comment.